pool-async/src/main/java/com/github/jasync/sql/db/pool/ActorBasedObjectPool.kt

```diff
@@ -257,6 +257,14 @@ private class ObjectPoolActor<T : PooledObject>(
             createObject(null)
             logger.trace { "scheduleNewItemsIfNeeded - creating new object ; $poolStatusString" }
         }
+
+        while (configuration.minObjects != null &&
+            (availableItems.size + inCreateItems.size) < configuration.minObjects &&
```

Comment:
I think `(availableItems.size + inCreateItems.size)` should be `totalItems` because we want to count also items in use.